example.env

```diff
@@ -1 +1,2 @@
-PORT=
\ No newline at end of file
+PORT=
+DEV_DB_URL=postgres://[username]:[password]@localhost:5432/[database]
```

Comment:
In example.env file just set the variable name without the value, (delete connection string)